src/timeMachine/actions/queries.ts

```diff
@@ -15,22 +15,16 @@ import {notify} from 'src/shared/actions/notifications'
 import {hydrateVariables} from 'src/variables/actions/thunks'
 
 // Constants
-import {
-  rateLimitReached,
-  resultTooLarge,
-  updateAggregateType,
-} from 'src/shared/copy/notifications'
+import {rateLimitReached, resultTooLarge} from 'src/shared/copy/notifications'
 
 // Utils
 import fromFlux from 'src/shared/utils/fromFlux'
 import {isFlagEnabled} from 'src/shared/utils/featureFlag'
 import {buildUsedVarsOption} from 'src/variables/utils/buildVarsOption'
 import {findNodes} from 'src/shared/utils/ast'
-import {isAggregateTypeError} from 'src/utils/aggregateTypeErrors'
```

Comment:
i just searched the codebase for `isAggregateTypeError` and only found it used here. Go ahead and delete `src/utils/aggregateTypeErrors` from since it's not being used anymore.